app/views/hub/clients/edit_take_action.html.erb

```diff
@@ -51,13 +58,12 @@
         :internal_note_body,
         t(".internal_note_body_label"),
         options: {rows: 8},
-        help_text: t(".blank_no_internal_note")
+        help_text: t(".blank_no_internal_note"),
       )%>
 
       <p><%=t(".send_clarifier")%></p>
-      <button class="button button--cta " type="submit">
-        <%= t("general.send") %>
-      </button>
+      <%= f.submit "Send", class: "button button--cta" %>
```

Comment:
Trivial: Consider using `t("general.send")` here. Hub i18n doesn't matter much, I admit. Not a blocker.